docs/api/web-contents.md

```diff
@@ -949,11 +949,17 @@ Emitted when a `<webview>` has been attached to this web contents.
 
 Returns:
 
-* `event` Event
-* `level` Integer - The log level, from 0 to 3. In order it matches `verbose`, `info`, `warning` and `error`.
-* `message` string - The actual console message
-* `line` Integer - The line number of the source that triggered this console message
-* `sourceId` string
+* `details` Event\<\>
+  * `message` string - Message text
+  * `level` string - Message severity
+    Possible values include `info`, `warning`, `error`, and `debug`.
```

Comment:
I think docs-parser will make this into a string enum if you format it similar to https://www.electronjs.org/docs/latest/api/app#appgetpathname

Reply:
The possible values list gets parsed and turned into a string enum which is confirmed in the [Artifact Comparison check.](https://github.com/electron/electron/pull/43617/checks)

```
level: ('info' | 'warning' | 'error' | 'debug');
```

https://github.com/electron/typescript-definitions/blob/28f88c950a524689d920a8e26698367102c171bb/src/utils.ts#L148-L149

Reply:
Yeah the docs parser is actually quite loose here, if you wanna see the cursed regex that pulls this off --> https://github.com/electron/docs-parser/blob/0a1bd843256995399b01d55862736e359462b8ae/src/markdown-helpers.ts#L456-L478